src/tutorials/0005-regular-files-api/07-exercise.md

```diff
@@ -14,6 +14,7 @@ Based on our file structure, the path could be expressed as:
 
 There are two valid ways to solve this challenge. Take your pick!
 
-**Hint:** In the code below we've already taken care of converting the `bufferedContents` to a string before returning the result, as you did yourself previously using `.toString()`.
-
-Don't forget to prepend `/ipfs/` to the IPFS Path string
+**Hints:**
+- In the code below we've already taken care of converting the `bufferedContents` to a string before returning the result, as you did yourself previously using `.toString()`.
+- Don't forget to prepend `/ipfs/` to the IPFS Path string
+- You need to concatenate all the data into a single buffer because the `ipfs.cat` method returns an `Async Iterable`. You can use the package `it-concat`.
```

Comment:
You never mentioned this in the lesson itself

Reply:
This was already addressed in Lesson 4. Should we add more than just the hint then?

Reply:
No, you're right, sorry. Just the hint is fine.